src/blocks/file-blocks/flat.tsx

```diff
@@ -19,43 +16,23 @@ export default function (props: FileBlockProps) {
           return acc;
         }, {});
       });
-      setModifiedData(csvData);
-      setIsDirty(false);
       return csvData;
     } catch (e) {
       return [];
     }
   }, [content]);
 
   return (
-    <div className="height-full d-flex flex-column">
-      <div className="flex-1" style={{ zIndex: 1 }}>
-        <Grid
-          data={modifiedData}
-          diffData={data}
-          canDownload={false}
-          isEditable
-          onEdit={(data: any) => {
-            setModifiedData(data);
-            setIsDirty(true);
-          }}
-        />
-      </div>
-      {isDirty && (
-        <button
-          className="position-absolute btn btn-primary inline-block"
-          style={{
-            bottom: "12px",
-            right: "175px",
-            zIndex: 10,
-          }}
-          onClick={() => {
-            onRequestUpdateContent(csvFormat(modifiedData));
-          }}
-        >
-          Save changes
-        </button>
-      )}
+    <div className="height-full d-flex flex-column flex-1">
+      <Grid
+        data={data}
+        diffData={data}
```

Comment:
hm it would be nice to preserve this. maybe blocks should get an `originalContent` prop in addition to `content`, to make it possible to show changes.

Reply:
oo good point! I like that.